lib/updateIndOk.ml

```diff
@@ -715,12 +715,13 @@ let reconstitute_person conf =
   (p, ext)
 
 let check_person conf base p =
-  match Update.check_missing_name base p with
-  | Some _ as err -> err
-  | None ->
-      Update.check_missing_witnesses_names conf
-        (fun e -> e.epers_witnesses)
-        p.pevents
+  let bind_none x f = match x with Some _ -> x | None -> f () in
+  let ( >>= ) = bind_none in
+  Update.check_missing_name base p >>= fun _ ->
```

Comment:
```suggestion
  Update.check_missing_name base p >>= fun () ->
```

otherwise, we could think that the function has an *interesting* unused parameter.